src/reference/asciidoc/redis.adoc

```diff
@@ -791,12 +791,104 @@ You should set the expiry at a large enough value to prevent this condition, but
 Starting with version 5.0, the `RedisLockRegistry` implements `ExpirableLockRegistry`, which removes locks last acquired more than `age` ago and that are not currently locked.
 
 
+[[redis-stream-outbound]]
+=== Redis Stream Outbound Channel Adapter
+
+Spring Integration 5.4 introduced Reactive Redis Stream outbound channel adapter to write Message payload into Redis stream.
+Outbound Channel adapter uses `ReactiveStreamOperations.add(...)` to send `Message` to stream.
+The following example shows how to use Java Configuration and Service class for Redis Stream Outbound Channel Adapter.
+
+====
+[source,java]
+----
+@Service
+public class MyService {
+
+    @Autowired
+    private FluxMessageChannel messageChannel;
+
+    public void sendMessage(Flux<Object> myFlux) {
+        messageChannel.subscribeTo(myFlux.map(myObj -> new GenericMessage<>(myObj)));
```

Comment:
This is too complicated in a simple configuration sample.
Why do we need this piece of code at all?
Why just show plain `ReactiveRedisStreamMessageHandler` with that `@ServiceActivator` not enough?
Making everything else implied.

Anyway: why stick only with `FluxMessageChannel`?
We really can't use any other channels?